db/migrate/20170403171417_rename_bet_to_bet_description_for_bet_model.rb

```diff
@@ -0,0 +1,5 @@
+class RenameBetToBetDescriptionForBetModel < ActiveRecord::Migration[5.0]
+  def change
+    rename_column :bets, :bet, :bet_description
```

Comment:
Since we are renaming things........ `bet.bet_description` could probably just be `bet.description`